_studio/mfx_lib/vpp/src/mfx_vpp_hw.cpp

```diff
@@ -3115,9 +3134,16 @@ mfxStatus VideoVPPHW::VppFrameCheck(
 #endif
     MFX_CHECK_STS(sts);
 
+    bool useCopyPassThrough = false;
+    if (true == m_config.m_bCopyPassThroughEnable && false == IsRoiDifferent(pTask->input.pSurf, pTask->output.pSurf))
```

Comment:
I don't like ```bool``` to ```bool``` conversions.
As for me
```
if (m_config.m_bCopyPassThroughEnable && !IsRoiDifferent(pTask->input.pSurf, pTask->output.pSurf))
```
is much better.
But I won't insist of changing this if you like your approach

Reply:
This is legacy. But I'm for the change, sure.

Reply:
Done.